components/tyk-enterprise-portal/values.yaml

```diff
@@ -32,12 +32,26 @@ license: ""
 #In case you want to deploy enterprise portal as a standalone app, you can configure a different dashboard URL using
 #the parameter bellow
 overrideTykDashUrl: ""
-
+# Enterprise portal can be deployed as StatefulSet or as Deployment
+kind: StatefulSet
 storage:
+  # User can set the storage type for portal.
+  # Supported types: fs, s3, db
+  type: "fs"
+  # Configuration values for using an SQL database as storage for Tyk Enterprise Portal
+  # In case you want to provide the connection string via secrets please
+  # refer to the existing secret inside the helmchart or the
+  # .Values.global.secrets.useSecretName variable
   database:
-    type: "sqlite3"
+    dialect: "sqlite3"
     connectionString: "db/portal.db"
+    enableLogs: true
```

Comment:
any specific reason to set it true by default? I am not much familiar with domain specific requirements but as far as i see from the docs, it is recommended to set it to false for prod environments. what do you think? 

https://tyk.io/docs/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal/configuration/#portal_database_enablelogs

Reply:
Comment addressed